### ISSUE TITLE
scss compile error using includeInProject and under TFS source control

### ISSUE THREAD

**Issue by MartijnSneijders** [madskristensen/WebCompiler#39]
Compiling file.scss files gives a dialog error saying something like "compilerconfig.json is wrong format". A unchecked-out file.css under source control (readonly file) seems to cause it.

Work around: setting all "includeInProjects" to "false" in compilerconfig.json, and removing all file.css from source control (TFS in my case). Or manually checking out **all** *.css and *.min.css in compilerconfig.json.

**Comment by madskristensen** [comment]
Does the output window show a more detailed error?

**Comment by MartijnSneijders** [comment]
Unauthorised access exception on File.WriteAll, because they are readonly.
I debugged it:
WebCompilerVsix.ProjectHelpers.CheckFileOutOfSourceControl(string file) gets called with a relative path (in my case "sass/file.css"). File.Exists("sass/file.css" fails. I guess the current directory is not set to my project base path? I changed the code to use an absolute path, this fixed it for me.

My change was this, in WebCompilerVsix.CompilerService.Processor:
_processor.BeforeProcess += (s, e) => { ProjectHelpers.CheckFileOutOfSourceControl(Path.Combine(e.BaseFolder, e.Config.OutputFile)); };
